README.md

```diff
@@ -243,6 +243,51 @@ Passwordless.configure do |config|
   config.sign_out_redirect_path = '/' # After a user signs out
 
   config.paranoid = false # Display email sent notice even when the resource is not found.
+
+  config.after_session_confirm = ->(session) {} # Called after a session is confirmed.
```

Comment:
```suggestion
  config.after_session_confirm = ->(request, session) {} # Called after a session is confirmed.
```

Let's just always send req + session in this. The other one is mostly there to keep the api backwards compatible.